### ISSUE TITLE
Build failure for ocamlfuse 2.7.1-cvs5 and conf-sqlite3 1

### ISSUE THREAD

**Issue by sunandosamaddar** [astrada/google-drive-ocamlfuse#514]
Hi, I'm getting this error when running `opam install google-drive-ocamlfuse` on a MBP 9,2:

```
The following actions will be performed:
  - install ocamlfuse              2.7.1-cvs5 [required by google-drive-ocamlfuse]
  - install conf-sqlite3           1          [required by sqlite3]
  - install sqlite3                4.4.1      [required by google-drive-ocamlfuse]
  - install google-drive-ocamlfuse 0.7.1
===== 4 to install =====
Do you want to continue? [Y/n] y

<><> Gathering sources ><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[google-drive-ocamlfuse.0.7.1] found in cache
[ocamlfuse.2.7.1-cvs5] found in cache
[sqlite3.4.4.1] found in cache

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[ERROR] The compilation of conf-sqlite3 failed at "/Users/sunandasamaddar/.opam/opam-init/hooks/sandbox.sh build pkg-config sqlite3".
[ERROR] The compilation of ocamlfuse failed at "/Users/sunandasamaddar/.opam/opam-init/hooks/sandbox.sh build ocaml setup.ml -build".

#=== ERROR while compiling ocamlfuse.2.7.1-cvs5 ===============================#
# context     2.0.2 | macos/x86_64 | ocaml-system.4.07.1 | https://opam.ocaml.org#50da06f4
# path        ~/.opam/default/.opam-switch/build/ocamlfuse.2.7.1-cvs5
# command     ~/.opam/opam-init/hooks/sandbox.sh build ocaml setup.ml -build
# exit-code   1
# env-file    ~/.opam/log/ocamlfuse-5276-6cc03f.env
# output-file ~/.opam/log/ocamlfuse-5276-6cc03f.out
### output ###
# [...]
# making ._d/Fuse_result.d from Fuse_result.ml
# ocamlc -c -cc "cc" -ccopt "-fPIC -D_FILE_OFFSET_BITS=64 -I. -pthread \
# 				-DPIC -DNATIVE_CODE -I/usr/local/include/osxfuse  \
# 				  -I/Users/sunandasamaddar/.opam/default/lib/camlidl  -o Fuse_bindings_stubs.o " Fuse_bindings_stubs.c 
# In file included from Fuse_bindings_stubs.c:17:
# ./Fuse_bindings.h:17:10: fatal error: 'fuse.h' file not found
# #include <fuse.h>
#          ^
# 1 error generated.
# make[1]: *** [Fuse_bindings_stubs.o] Error 2
# make: *** [native-code-library] Error 2
# E: Failure("Command 'make -C lib INCDIRS=/Users/sunandasamaddar/.opam/default/lib/camlidl' terminated with error code 2")


#=== ERROR while compiling conf-sqlite3.1 =====================================#
# context     2.0.2 | macos/x86_64 | ocaml-system.4.07.1 | https://opam.ocaml.org#50da06f4
# path        ~/.opam/default/.opam-switch/build/conf-sqlite3.1
# command     ~/.opam/opam-init/hooks/sandbox.sh build pkg-config sqlite3
# exit-code   1
# env-file    ~/.opam/log/conf-sqlite3-5276-d8a4ae.env
# output-file ~/.opam/log/conf-sqlite3-5276-d8a4ae.out



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - build conf-sqlite3 1
| - build ocamlfuse    2.7.1-cvs5
+- 
- No changes have been performed

The packages you requested declare the following system dependencies. Please make sure they are installed before retrying:
    sqlite3


```
How do I proceed from here? I tried installing sqlite3 and ocaml-sqlite3 using MacPorts, but whether these 2 packages are installed seems to have no effect.

**Comment by astrada** [comment]
You need these external packages: `osxfuse`, `gmp`, `lzlib`, `pkg-config`, `sqlite3`. With homebrew, you can use these commands: 
```sh
opam install depext
opam depext google-drive-ocamlfuse
```
Here you can find Travis build logs for macos: https://travis-ci.org/ocaml/opam-repository/jobs/456680431

**Comment by sunandosamaddar** [comment]
Hi @astrada, hearty new year wishes! Here's my observation. I did `port install osxfuse gmp pkgconfig sqlite3` and followed steps in `INSTALL` after unpacking `lzlib-1.10.tar.gz` ([home](https://www.nongnu.org/lzip/lzlib.html)). `opam install google-drive-ocamlfuse` did not pick up the default `sqlite3` system installation so I used `port install`. However, now there's still a build failure just for `ocamlfuse 2.7.1-cvs5` for the same error shown in above snippet. `osxfuse` is installed.
